test/passes/simplify-globals_all-features_fuzz-exec.txt

```diff
@@ -4,12 +4,12 @@
  (type $0 (func (param f32 i31ref i64 f64 funcref)))
  (type $1 (func (result funcref)))
  (global $global$0 (mut funcref) (ref.null nofunc))
- (elem declare func $0)
- (export "export" (func $1))
+ (elem $decl declare func$0)
```

Comment:
Why is this now named?

Reply:
This is an output .txt file, so the printer now names this. We don't need to name declarative segments in the input.

Reply:
Can we avoid printing this name in the output, if it's unnecessary? It just seems to add clutter compared to before.

Reply:
The auto update script only recognizes named items, so even with --all-items, it won't include these declarative segments in the output unless we give them names. It previously included them because it thought "declare" was its name.

Are you ok with dropping declarative element segments from generated test output? If so, I can remove this emitted name. Otherwise we can consider trying to change how the update script works, but that is very difficult.

Reply:
I'm confused, didn't that work before this PR? We didn't have named declarative segments before, so it seems like it must have?